src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1237,30 +1237,60 @@ static std::vector<TestCollection> testSuite = {
   { create_init_test("SC version", "Check if SC firmware is up-to-date", ""), scVersionTest },
   { create_init_test("Verify kernel", "Run 'Hello World' kernel test", "verify.xclbin"), verifyKernelTest },
   { create_init_test("DMA", "Run dma test", "verify.xclbin"), dmaTest },
-  { create_init_test("iops", "Run scheduler performance measure test", "verify.xclbin"), iopsTest },
+  { create_init_test("scheduler-perf-test", "Run scheduler performance measure test", "verify.xclbin"), iopsTest },
   { create_init_test("Bandwidth kernel", "Run 'bandwidth kernel' and check the throughput", "bandwidth.xclbin"), bandwidthKernelTest },
   { create_init_test("Peer to peer bar", "Run P2P test", "bandwidth.xclbin"), p2pTest },
   { create_init_test("Memory to memory DMA", "Run M2M test", "bandwidth.xclbin"), m2mTest },
   { create_init_test("Host memory bandwidth test", "Run 'bandwidth kernel' when host memory is enabled", "bandwidth.xclbin"), hostMemBandwidthKernelTest },
-  { create_init_test("bist", "Run BIST test", "verify.xclbin", true), bistTest },
-  { create_init_test("vcu", "Run decoder test", "transcode.xclbin"), vcuKernelTest }
+  { create_init_test("hw-scheduler-bist", "Run BIST test", "verify.xclbin", true), bistTest },
+  { create_init_test("video-decoder-test", "Run decoder test", "transcode.xclbin"), vcuKernelTest }
 };
 
+static std::vector<std::string>
+get_run_test_names(const std::string& long_test_name)
+{
+  static std::map<std::string, std::string> long_name_to_short_name={
+      { "Aux connection",                "aux-connection"    },
+      { "PCIE link",                     "pcie-link"   },
+      { "SC version",                    "sc-version"   },
+      { "DMA",                           "dma"  },
+      { "Verify kernel",                 "verify"},
+      { "scheduler-perf-test",           "iops"  },
+      { "Bandwidth kernel",              "mem-bw"},
+      { "Peer to peer bar",              "p2p"},
+      { "Memory to memory DMA",          "m2m"},
+      { "Host memory bandwidth test",    "hostmem-bw"},
+      { "hw-scheduler-bist",             "bist"},
+      { "video-decoder-test",            "vcu"}
+  };
+
+  std::vector<std::string> testNames;
+
+  if (long_name_to_short_name.find(long_test_name) != long_name_to_short_name.end()){
```

Comment:
_Coding recommendation:_ when an if statement is 1 line, you don't have to use {}